src/features/controls.ts

```diff
@@ -356,32 +382,32 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
           // again for this particular <model-element> instance:
           this[$userPromptedOnce] = true;
           this[$waitingToPromptUser] = false;
-          this[$promptElementVisible] = true;
+          this[$promptElementVisibleTime] = time;
 
           this[$promptElement].classList.add('visible');
         }
       }
 
-      if (this[$promptElementVisible] &&
+      if (isFinite(this[$promptElementVisibleTime]) &&
           this.interactionPromptStyle === InteractionPromptStyle.WIGGLE) {
-        const transformString =
-            self.getComputedStyle(this[$promptAnimatedContainer])
-                .getPropertyValue('transform');
-        // Parse the fifth term of computed transform style
-        // which is in form of matrix(n0, n1, n2, n3, n4, n5)
-        // @see https://www.w3.org/TR/css-transforms-1/#serialization-of-the-computed-value
-        const offset = parseFloat(transformString.split(',')[4]);
-        const delta = offset - this[$lastPromptOffset];
-
-        if (isFinite(offset)) {
-          const scene = this[$scene];
-
-          this[$lastPromptOffset] = offset;
-          scene.setPivotRotation(
-              scene.getPivotRotation() +
-              delta / scene.width * OFFSET_ROTATION_MULTIPLIER);
-          this[$needsRender]();
-        }
+        const scene = this[$scene];
+        const animationTime =
+            ((time - this[$promptElementVisibleTime]) % PROMPT_ANIMATION_TIME) /
+            PROMPT_ANIMATION_TIME;
+        const offset = wiggle(animationTime);
+        const opacity = fade(animationTime);
+
+        const xOffset = offset * scene.width * 0.05;
+        const deltaTheta = (offset - this[$lastPromptOffset]) * Math.PI / 8;
+
+        this[$promptAnimatedContainer].style.transform =
+            `translateX(${xOffset}px)`;
+        this[$promptAnimatedContainer].style.opacity = `${opacity}`;
+
+        this[$controls].adjustOrbit(deltaTheta * Math.PI / 6, 0, 0, 0);
```

Comment:
What's the difference here with `Math.PI / 6` vs `Math.PI / 8` above?

Reply:
That's a bug! Will fix.